samvera-labs/bulkrax
# Exporter export sources

**Work completed:** 

- add javascript for dynamic selection of export source
- add work_type to export form
- add a method to the parser to create from work type
- added i18n support for export#edit form
- added collection autosuggest
- store id of collection from autosuggest
- tests
- some additonal js stuff (see comments in exporter.js)
- ensure limit works
- ensure form fields are populated correctly on edit
- input styling 

Once merged, should also resolve #169, #170, and #171
